src/com/rabbitmq/client/impl/recovery/AutorecoveringConnection.java

```diff
@@ -181,6 +188,10 @@ public boolean isOpen() {
      * @see com.rabbitmq.client.Connection#close()
      */
     public void close() throws IOException {
+		synchronized(recoveryLock)
```

Comment:
I'm pretty sure this indentation is not what the rest of the codebase (or even this particular class) uses.